docs/application/dotnet/guides/nlp/nlp.md

```diff
@@ -0,0 +1,223 @@
+
+-   Language Detect support
+
+    You can get language from a sentence, the type of return is LanguageDetectedResult.
+
+## Word Tokenize
+This method break up the sentence into words and punctuation.
+
+
+## Part of Speech
+This method breaks up the sentence into words and punctuation with tags attributes.
+
+
+## Named Entity Detection
+This method breaks up the sentence into words and punctuation with tags attributes.
+
+
+## Language Detect
+This method is used to detect the language of a sentence.
+
+
+## Prerequisites
+
+Enable your application to use the Nlp functionality:
+
+1.  Using the [Tizen.Nlp](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Nlp.html) namespace, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+       <privilege>http://tizen.org/privilege/datasharing</privilege>
+    </privileges>
+    ```
+
+2.  Using the methods and properties of the `Tizen.Nlp` namespace, include it in your application:
+
+    ```
+    using Tizen.Nlp;
+    ```
+
+## Receive Tokens and Tags - WordTokenize
+
+receive the tokens from sentence:
+
+1.  Construct NaturalLanguageProcess on OnCreate(), and connect the Nlp service on init of app:
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the WordTokenizeAsync(string msg) in a async task method:
+
+    ```
+    public async Task OnTokenButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.WordTokenizeAsync(msg);
+        WordTokenizeResult wr = await ret;
+        ..
+    }
+    ```
+
+3.  When nlp object is no longer needed, call Dispose() to release the resource of Nlp:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+
+## Receive Tokens and Tags - PosTag
+
+receive the tokens and tags from sentence:
+
+1.  Construct NaturalLanguageProcess on OnCreate(), and connect the Nlp service on init of app:
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the PosTagAsync(string msg) in a async task method:
+
+    ```
+    public async Task OnPosButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.PosTagAsync(msg);
+        PosTagResult pr = await ret;
+        ..
+    }
+    ```
+
+3.  When nlp object is no longer needed, call Dispose() to release the resource of Nlp:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+
+## Receive Tokens and Tags - Named Entity
+
+receive the tokens and tags from sentence:
```

Comment:
Receive

@divyk, why are you breaking grammar?
Please start setences with capital letters.